prboom2/src/w_mmap.c

```diff
@@ -128,9 +129,11 @@ void W_InitCache(void)
 #endif
       if (!mapped_wad[wad_index].data)
       {
-        mapped_wad[wad_index].hnd = CreateFile(wadfiles[wad_index].name,
+        wchar_t *wname = dsda_ConvertUtf8ToWide(wadfiles[wad_index].name);
```

Comment:
What decides when this conversion needs to happen and when it doesn't? Why in this file but not for instance when opening a demo file?

Reply:
This conversion is needed only for Win API functions, `CreateFileW()` in this case.

Reply:
Ok thanks